app/assets/stylesheets/application.scss

```diff
@@ -73,21 +78,62 @@ div#face-header {
 }
 
 .icons a.facebook {
-background: url(./facebook.png)
-left top no-repeat;
+  background: url(./facebook.png)
+  left top no-repeat;
 }
 .icons a.github {
-background: url(./github.png)
-left top no-repeat;
+  background: url(./github.png)
+  left top no-repeat;
 }
 .icons a.googleplus {
-background: url(./googleplus.png)
-left top no-repeat;
+  background: url(./googleplus.png)
+  left top no-repeat;
 }
 .icons a.email {
-background: url(./email.png)
-left top no-repeat;
+  background: url(./email.png)
+  left top no-repeat;
 }
 .icons a:hover {
-background-position: left -34px;
+  background-position: left -34px;
+}
+
+.racing-stripe {
+  z-index: 1;
+  left: 20px;
+}
+#fat-stripe {
+  position: absolute;
+  width: 5vw;
+  height: 150%;
+  background-color: rgba(209, 0, 8, 0.57);
+  top: 0;
+  left: 3.5vw;
+}
+
+#thin-stripe {
+  position: absolute;
+  height: 150%;
+  top: 0px;
+  width: 1.5vw;
+  background-color: rgba(209, 0, 8, 0.57);
+  left: .5vw;
+}
+
+nav {
+  z-index: 2;
+}
+
+.navbar-inverse {
+  background-color: #D10008;
```

Comment:
Color literals like `#D10008` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#D10008` should be written as `#d10008`